projects/kit/styles/components/progress-bar.less

```diff
@@ -104,6 +104,18 @@
         border-radius: inherit;
     }
 
+    &[tuiProgressFixedGradient] {
+        .progressValue({
+            border-radius: inherit;
```

Comment:
We already set this property above
https://github.com/taiga-family/taiga-ui/blob/01967d9b299576307478a89a4ab6f697ef5c4052/projects/kit/styles/components/progress-bar.less#L57

Why should we set it again ?